src/app/parcels/form/ParcelForm.tsx

```diff
@@ -124,20 +130,33 @@ const mergeDateAndTime = (date: string, time: string): Dayjs => {
     return dayjs(date).hour(dayjsTime.hour()).minute(dayjsTime.minute());
 };
 
+const parcelModalRouterPath = (parcelId: string): string => `/parcels?parcelId=${parcelId}`;
+
+const databaseErrorMessageFromErrorType = (
```

Comment:
Exported error message calculation to separate function - hopefully this should be neater :)